### PR TITLE
Add un7zip library

### PR DESCRIPTION
**Pull request by bucanero**
ported a simple library to extract `.7z` files

### CODE REVIEW

File: libun7zip/PSPBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+arch=('mips')
+license=('Apache-2.0 license')
+url="https://github.com/bucanero/libun7zip"
+makedepends=()
+source=("git+https://github.com/bucanero/${pkgname}#commit=23e7cecd979fd893a6a7d3fcc51b2478cd860607")
+sha256sums=('SKIP')
+
+build() {
+  cd $pkgname
+  make -f Makefile.psp install
```
Comment (sharkwouter) [inline libun7zip/PSPBUILD:14]: This command should probably not contain install. We do not want the user's system to be modified by a PSPBUILD.

**Review by sharkwouter — CHANGES_REQUESTED** [review]
This is cool, thanks! The PSPBUILD looks good other than the one remark I had. The package which was build looks good as well.

File: libun7zip/PSPBUILD (hunk 1 above)
Comment (bucanero) [inline libun7zip/PSPBUILD:14]:
```suggestion
  make -f Makefile.psp
```

Comment (bucanero) [reply libun7zip/PSPBUILD]: Removed `install`, it should be fixed now.

**Comment by sharkwouter** [comment]
Thanks for the contribution!

**Pull request merged**